bundles/paikkatietoikkuna/coordinatetransformation/components/CoordinateSystemSelection.js

```diff
@@ -0,0 +1,68 @@
+Oskari.clazz.define('Oskari.coordinatetransformation.component.CoordinateSystemSelection',
+    function (view) {
+        this.instance = view;
+        this.loc = view.loc;
+        this.element = null;
+        this.select = Oskari.clazz.create('Oskari.coordinatetransformation.component.select', view );
+        this.selectInstance = null;
+        this.dropdowns = null;
+        this._template = {
+            systemWrapper: jQuery('<div class="coordinateSystemWrapper"></div>'),
+            coordinatSystemSelection: _.template(
+                '<div class="transformation-system">' +
+                    '<h5><%= title %></h5>'+
```

Comment:
Elsewhere in Oskari placeholders for lodash-templates have been marked with syntax like ${title}. Both work so it's not a biggie. Just pointing this out so we could be consistent.